Simplified/Log.swift

```diff
@@ -44,25 +45,10 @@ final class Log: NSObject {
     // Format string
     let formattedMsg = "[\(levelToString(level))] [\(timestamp)] \(tag): \(message)\(error == nil ? "" : "\n\(error!)")\n"
     
-    // Write to console
-    NSLog(formattedMsg)
-    
-    // Write to file
-    var overwrite = false
-    if let size = try? FileManager.default.attributesOfItem(atPath: logUrl.path)[FileAttributeKey.size] as! Int {
-      if size > 1048576 {
-        overwrite = true
-      }
-    }
-    if overwrite {
-      try? formattedMsg.write(to: logUrl, atomically: false, encoding: .utf8)
+    if level != .debug {
+      CLSNSLogv("%@", getVaList([formattedMsg]))
```

Comment:
According to the docs in CLSLogging.h  it seems like we should actually not use this in production. I had not seen this before. Can you add a `#if DEBUG` to use that only when debugging and use `CLSLogv` otherwise?